app/templates/loading.hbs

```diff
@@ -8,28 +8,25 @@
         </button>
       </div>
 
-      {{#unless features.enterpriseVersion}}
-        {{#if auth.signedIn}}
-
-          {{broadcast-tower toggleBroadcasts="toggleBroadcasts" status=broadcasts.lastBroadcastStatus count=broadcasts.length}}
+      {{#if (and features.broadcasts auth.signedIn)}}
+        {{broadcast-tower toggleBroadcasts="toggleBroadcasts" status=broadcasts.lastBroadcastStatus count=broadcasts.length}}
 
-          <ul class="broadcasts {{if showBroadcasts 'is-open'}}">
-            {{#each broadcasts.content as |broadcast|}}
-              <li>
-                <p>
-                  <span class="broadcast-status {{broadcast.category}}" title="Transmitted on {{broadcast.updated_at}}"></span>
-                  <span class='message'>{{{broadcast.message}}}</span>
-                  <button {{action 'markBroadcastAsSeen' broadcast}} title="hide this broadcast" class="broadcast-close no-button" type="button">
-                    {{svg-jar 'icon-failed' class="icon"}}
-                  </button>
-                </p>
-              </li>
-            {{else}}
-              <li><p>There are no broadcasts</p></li>
-            {{/each}}
-          </ul>
-        {{/if}}
-      {{/unless}}
+        <ul class="broadcasts {{if showBroadcasts 'is-open'}}">
+          {{#each broadcasts.content as |broadcast|}}
+            <li>
+              <p>
+                <span class="broadcast-status {{broadcast.category}}" title="Transmitted on {{broadcast.updated_at}}"></span>
+                <span class='message'>{{{broadcast.message}}}</span>
+                <button {{action 'markBroadcastAsSeen' broadcast}} title="hide this broadcast" class="broadcast-close no-button" type="button">
+                  {{svg-jar 'icon-failed' class="icon"}}
+                </button>
+              </p>
+            </li>
+          {{else}}
+            <li><p>There are no broadcasts</p></li>
+          {{/each}}
+        </ul>
+      {{/if}}
```

Comment:
wow, I don’t even know about this template, this duplication should probably be extracted? But that’s orthogonal to this PR.

Reply:
Yeah, for sure. The loading templates definitely need to be consolidated.